### PR TITLE
Added content for hero section

### PR DESCRIPTION
**Pull request by ishi0301**

### CODE REVIEW

**Comment by saviourcode** [comment]
Issue 1, The first section got padding of right,



How it supposed to be,

**Comment by saviourcode** [comment]
Issue 2,
For a mobile view that is a Screen width of less than 770px, the whole section overflows to top and bottom,

**Comment by saviourcode** [comment]
@ishi0301 Please correct the above two issues.

File: style.css [hunk 1]
```diff
@@ -294,7 +294,8 @@ input:checked + label:after {
   height: 100vh;
   min-height: 520px;
   position: relative;
-  left:-15px;
+  left:-15px; 
+  width: 80%;
```
Comment (saviourcode) [inline style.css:298]: Remove this width property

File: style.css [hunk 2]
```diff
@@ -442,10 +443,41 @@ input:checked + label:after {
 }
 
 .hero-container h1{
-  font-size: 6vw;
+  font-size: 20px;
   margin-bottom: 5vh;
 
 }
+.container .box {
```
Comment (saviourcode) [inline style.css:450]: Make this section responsive

**Review by sjchoure — CHANGES_REQUESTED** [review]

File: style.css (hunk 1 above)
Comment (saviourcode) [reply style.css:298]: It will resolve issue 1

**Comment by ishi0301** [comment]
@saviourcode Please check.

**Review by sjchoure — APPROVED** [review]
LGTM! Awesome Work!! @ishi0301

**Comment by ishi0301** [comment]
Thanks @saviourcode

**Pull request merged**